docs/root/configuration/overview/v2_overview.rst

```diff
@@ -0,0 +1,508 @@
+<config_overview_lds>` or :ref:`CDS <config_cluster_manager_cds>` configured in
+:ref:`dynamic_resources <envoy_api_field_Bootstrap.dynamic_resources>`.
+
+Example
+-------
+
+Below we will use YAML representation of the config protos and a running example
+of a service proxying HTTP from 127.0.0.1:10000 to 127.0.0.2:1234.
+
+Static
+^^^^^^
+
+A minimal fully static bootstrap config is provided below:
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 127.0.0.1, port_value: 9901 }
+
+  static_resources:
+    listeners:
+    - name: listener_0
+      address:
+        socket_address: { address: 127.0.0.1, port_value: 10000 }
+      filter_chains:
+      - filters:
+        - name: envoy.http_connection_manager
+          config:
+            stat_prefix: ingress_http
+            codec_type: AUTO
+            route_config:
+              name: local_route
+              virtual_hosts:
+              - name: local_service
+                domains: ["*"]
+                routes:
+                - match: { prefix: "/" }
+                  route: { cluster: some_service }
+            http_filters:
+            - name: envoy.router
+    clusters:
+    - name: some_service
+      connect_timeout: 0.25s
+      type: STATIC
+      lb_policy: ROUND_ROBIN
+      hosts: [{ socket_address: { address: 127.0.0.2, port_value: 1234 }}]
+
+Mostly static with dynamic EDS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A bootstrap config that continues from the above example with :ref:`dynamic endpoint
+discovery <arch_overview_dynamic_config_sds>` via an EDS gRPC management server listening
+on 127.0.0.3:5678 is provided below:
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 127.0.0.1, port_value: 9901 }
+
+  static_resources:
+    listeners:
+    - name: listener_0
+      address:
+        socket_address: { address: 127.0.0.1, port_value: 10000 }
+      filter_chains:
+      - filters:
+        - name: envoy.http_connection_manager
+          config:
+            stat_prefix: ingress_http
+            codec_type: AUTO
+            route_config:
+              name: local_route
+              virtual_hosts:
+              - name: local_service
+                domains: ["*"]
+                routes:
+                - match: { prefix: "/" }
+                  route: { cluster: some_service }
+            http_filters:
+            - name: envoy.router
+    clusters:
+    - name: some_service
+      connect_timeout: 0.25s
+      lb_policy: ROUND_ROBIN
+      type: EDS
+      eds_cluster_config:
+        eds_config:
+          api_config_source:
+            api_type: GRPC
+            cluster_name: [xds_cluster]
+    - name: xds_cluster
+      connect_timeout: 0.25s
+      type: STATIC
+      lb_policy: ROUND_ROBIN
```

Comment:
nit: When using api_type as `GRPC`, the `xds_cluster` should have `http2_protocl_options` otherwise EDS in not working.